syntax = "proto3";

service BookService {
  rpc AddBook (BookRequest) returns (BookResponse);
  rpc GetBook (BookIdRequest) returns (BookResponse);
}

message BookRequest {
  int32 id = 1;
  string title = 2;
  string author = 3;
}

message BookIdRequest {
  int32 id = 1;
}

message BookResponse {
  int32 id = 1;
  string title = 2;
  string author = 3;
}
